exporters/elasticsearch-exporter/README.md

```diff
@@ -19,6 +19,7 @@ those as templates in this module's resources folder.
 > **Note:** the indexes are created as required, and will not be created twice if they already exist. However,
 > once disabled, they will not be deleted, that is up to the administrator. Similarly, data is never deleted by
 > the exporter, and must, again, be deleted by the administrator when it is safe to do so.
+> A [retention](#retention) policy can be configured to automatically delete data after a certain number of days.
```

Comment:
🔧 We have an still open PR which will move this README to the official docs and replace it here with a link to the docs. We must make sure to include this configuration in that PR.